sdk/samples/iot/provisioning/src/paho_iot_provisioning_example.c

```diff
@@ -77,12 +77,20 @@ static int connect_device();
 static int subscribe();
 
 //
-// Messaging functions
+// Device registration functions
 //
 static int register_device();
 static int get_operation_status();
+static az_result parse_operation_message(
+    char * topic,
+    int topic_len,
+    MQTTClient_message const * message,
```

Comment:
> * [](start = 28, length = 2)

remove this space between type and *.

When defining pointer, code style is following:

type* var;

Reply:
Fix it all places

---
In reply to: [451026524](https://github.com/Azure/azure-sdk-for-c/pull/883#discussion_r451026524) [](ancestors = 451026524)

Reply:
forgot to run clang-format.  thanks